src/mainwindow.cpp

```diff
@@ -34,6 +34,7 @@
 #include <QWheelEvent>
 #include <QMouseEvent>
 #include <QTimer>
+#include <QScreen>
```

Comment:
Would you please also remove `#include <QDesktopWidget>` (a few lines below)?

Reply:
sure. completely forgot it.